pkg/notifications/email_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notifications
+
+import (
+	"context"
+	"testing"
+
+	"github.com/sigstore/rekor-monitor/pkg/identity"
+	"github.com/wneessen/go-mail"
+)
+
```

Comment:
It might be a bit overkill, but can we try adding a unit test with a mocked SMTP server like https://github.com/mocktools/go-smtp-mock?

Reply:
should be implemented!